tests/pkg_u8g2/Makefile

```diff
@@ -59,4 +59,7 @@ CFLAGS += -DTEST_PIN_RESET=$(TEST_PIN_RESET)
 
 CFLAGS += -DTEST_DISPLAY=$(TEST_DISPLAY)
 
+# HACK Blacklist native as `murdock` fails on utf-8 characters for native tests
+TEST_ON_CI_WHITELIST += nrf52dk samr21-xpro
```

Comment:
Does it make sense to whitelist.  What if other boards are added in (I think there is a pr for the esp)...  Why not just use a blacklist?

Reply:
Because this is currently not supported as `TEST_ON_CI_BLACKLIST` does not exist.

If you want I can split the variable introduction out of
* makefiles/murdock.inc.mk: change policy to run tests by default #11680